### PR TITLE
Set up automatic mysql

### PR DESCRIPTION
**Pull request by cal27279**
Hi @catlook! Could you please review and edit this PR when you have a chance? Thank you =)
I'm retiring and redirecting an article.

### CODE REVIEW

**Review by catlook — APPROVED** [review]
Golden!

**Comment by catlook** [comment]
@cal27279, good to go!

File: static/_redirects [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # Redirects
 
+/support/how-to/setup-automatic-mysql-server-backups-to-cloud-files-from-windows/  /how-to/setup-automatic-mysql-server-backups-to-cloud-files-from-windows/ 404
+/how-to/setup-automatic-mysql-server-backups-to-cloud-files-from-windows/  /support/how-to/setup-automatic-mysql-server-backups-to-cloud-files-from-windows/  301
```
Comment (stephamon) [inline static/_redirects:4]: This redirect looks like it's pointing back to itself. I think the retired articles are in a section further down the file and are formatted a little differently.

Comment (cal27279) [reply static/_redirects:4]: :eyes: on it

**Review by stephamon — APPROVED** [review]

**Pull request merged**